src/content/en/fundamentals/_redirects.yaml

```
@@ -130,6 +130,9 @@ redirects:
 - from: /web/fundamentals/feed.xml
   to: /web/fundamentals/rss.xml
 
+- from: /web/fundamentals/payments/payment-apps-developer-guide/android-payment-apps/
```

```suggestion
- from: /web/fundamentals/payments/payment-apps-developer-guide/android-payment-apps
```

This is a file, not a directory.